contracts/child/ChildERC721.sol

```diff
@@ -1,12 +1,11 @@
 pragma solidity ^0.4.24;
 
-import { ERC721Full } from "openzeppelin-solidity/contracts/token/ERC721/ERC721Full.sol";
-
+import "./ERC721.sol";
 import "./ChildToken.sol";
 import "./IParentToken.sol";
 
 
-contract ChildERC721 is ChildToken, ERC721Full {
+contract ChildERC721 is ChildToken, ERC721 {
```

Comment:
Use `ERC721Full` and remove explicit `_transferFrom`.

Reply:
cc @0xAshish @atvanguard

Reply:
this one is old PR, new one which got merged has ERC721Full.